api/specs.pb.go

```diff
@@ -2064,6 +2085,25 @@ func (m *ContainerSpec) MarshalTo(dAtA []byte) (int, error) {
 		i++
 		i = encodeVarintSpecs(dAtA, i, uint64(m.PidsLimit))
 	}
+	if len(m.Sysctls) > 0 {
```

Comment:
~this check looks redundant; the code inside the `for` loop won't be executed if it's empty; https://play.golang.org/p/SkHL_G_iJtv~


edit: nevermind. Didn't notice I was looking at generated code ("Code generated by protoc-gen-gogo. DO NOT EDIT.") :blush: